packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1547,6 +1558,20 @@ class EditableText extends StatefulWidget {
   /// {@macro flutter.services.TextInputConfiguration.enableIMEPersonalizedLearning}
   final bool enableIMEPersonalizedLearning;
 
+  /// {@template flutter.widgets.EditableText.spellCheckConfiguration}
+  /// Configuration that details how spell check should be performed.
+  ///
+  /// Specifies the [SpellCheckService] used to spell check text input and the
+  /// [TextStyle] used to style text with misspelled words.
+  ///
+  /// If the [SpellCheckService] is left null, spell check is disabled by
+  /// default unless the [DefaultSpellCheckService] is used, in which case it is
+  /// used. It is currently supported only on Android.
```

Comment:
I'm slightly confused by this paragraph.  Is this correct?

If the [SpellCheckService] is left null, spell check is disabled by default except on Android, where the [DefaultSpellCheckService] will be used.

Reply:
Or maybe your original text should just be changed like this and then it's fine? "[DefaultSpellCheckService] is used" => "[DefaultSpellCheckService] is supported"